### PR TITLE
Fix deprecated method delegation in tracer.rb for Ruby 3.0

### LINKED ISSUE #3849: `bundle exec srb init` crashes in Ruby 3.0.0
#### Input

`.ruby-version`

```text
3.0.0
```

`Gemfile`

```ruby
source 'https://rubygems.org'

gem 'sorbet
```

`test.rb`

```ruby
class A
end
```

#### Observed output

Run `bundle install` and then

```shell
$ bundle exec srb init

Hey there!

This script will get this project ready to use with Sorbet by creating a
sorbet/ folder for your project. It will contain:

- a config file
- a bunch of 'RBI files'

RBI stands for 'Ruby Interface'; these files define classes, methods, and
constants that exist, but which Sorbet doesn't always know about.

Heads up:

To set up your project, this script will take two potentially destructive
actions:

1.  It will require every file in your project. Specifically, every script in
    your project will be run, unless that script checks if __FILE__ == $PROGRAM_NAME
    before running any code, or has the magic comment # typed: ignore in it.

2.  It will add a comment to the top of every file (like # typed: false or
    # typed: true, depending on how many errors were found in that file.)

Would you like to continue? [Y/n] 
Generating: sorbet/config
Reusing existing config file: sorbet/config
Generating: sorbet/rbi/sorbet-typed/
Generating: sorbet/rbi/gems/
LoadError: cannot load such file -- sorbet
LoadError: cannot load such file -- sorbet-static
/home/dee-see/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6111/lib/gem-generator-tracepoint/tracepoint_serializer.rb:35:in `initialize': wrong number of arguments (given 1, expected 0; required keywords: files, delegate_classes) (ArgumentError)
	from /home/dee-see/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6111/lib/gem-generator-tracepoint/tracer.rb:40:in `new'
	from /home/dee-see/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6111/lib/gem-generator-tracepoint/tracer.rb:40:in `new'
	from /home/dee-see/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6111/lib/gem-generator-tracepoint.rb:43:in `main'
	from /home/dee-see/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6111/bin/srb-rbi:232:in `block in make_step'
	from /home/dee-see/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6111/bin/srb-rbi:118:in `init'
	from /home/dee-see/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6111/bin/srb-rbi:196:in `main'
	from /home/dee-see/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6111/bin/srb-rbi:237:in `<main>'
```

#### Expected behavior

It shouldn't crash :)

- - -

It works as expected in 2.7.2

I really expected this issue to exist and I looked for duplicates but couldn't find any. Sorry if this is known!

### PR DESCRIPTION
**Pull request by connorshea**
Use `ruby2_keywords` to support Ruby 2.6, 2.7, _and_ 3.0.

See the "compatible delegation" section in this blog post for more information about this solution: https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/

Essentially, Ruby changed the way it handles arguments in method definitions, and this was causing deprecation warnings in Ruby 2.7.2, which became an error with the release of Ruby 3.

I don't love that this defines `ruby2_keywords` in the main namespace like this, so I'd be open to feedback as to how this should be implemented. We could use [the ruby2_keywords gem](https://github.com/ruby/ruby2_keywords), but I'm hesitant to add an extra dependency to Sorbet.

Sorbet still fails to generate the hidden definitions on Ruby 3.0 with this change, but it makes it farther into the process (it hits an error about sorted_set since it was extracted into a separate gem? Need to look into that more)

### Motivation

Fixes #3849 so that Ruby 3.0 can be used (well, partially).

### Test plan

This _should_ be covered by the existing hidden-definitions tests. I've run the hidden definitions generator for my Rails app [vglist](https://github.com/connorshea/vglist) on Ruby 2.6, and 2.7 and verified that it works for both, and resolves the deprecation warning in Ruby 2.7. For 3.0, it fixes the initial problem, but another issue occurs later in the hidden definitions generation process that seems unrelated, so it still doesn't fully work.

### CODE REVIEW

**Comment by connorshea** [comment]
Before this change on Ruby 3:

```
Connors-MacBook-Pro-2:vglist connorshea$ CI=true bundle exec srb rbi hidden-definitions
Running command: hidden-definitions
Generating: sorbet/rbi/hidden-definitions/
Requiring all of your code
/Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/cache.rb:10:in `initialize': wrong number of arguments (given 3, expected 2) (ArgumentError)
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/gem-generator-tracepoint/tracer.rb:40:in `new'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/gem-generator-tracepoint/tracer.rb:40:in `new'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache.rb:45:in `setup'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap.rb:26:in `setup'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/setup.rb:31:in `<top (required)>'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/activesupport-6.1.0/lib/active_support/dependencies.rb:332:in `require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/activesupport-6.1.0/lib/active_support/dependencies.rb:332:in `block in require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/activesupport-6.1.0/lib/active_support/dependencies.rb:299:in `load_dependency'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/activesupport-6.1.0/lib/active_support/dependencies.rb:332:in `require'
	from /Users/connorshea/Programming/vglist/config/boot.rb:5:in `<top (required)>'
	from /Users/connorshea/Programming/vglist/config/application.rb:2:in `require_relative'
	from /Users/connorshea/Programming/vglist/config/application.rb:2:in `<top (required)>'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/activesupport-6.1.0/lib/active_support/dependencies.rb:332:in `require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/activesupport-6.1.0/lib/active_support/dependencies.rb:332:in `block in require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/activesupport-6.1.0/lib/active_support/dependencies.rb:299:in `load_dependency'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/activesupport-6.1.0/lib/active_support/dependencies.rb:332:in `require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/require_everything.rb:25:in `load_rails'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/require_everything.rb:18:in `require_everything'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/hidden-definition-finder.rb:64:in `require_everything'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/hidden-definition-finder.rb:43:in `main'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/hidden-definition-finder.rb:38:in `main'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/bin/srb-rbi:232:in `block in make_step'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/bin/srb-rbi:224:in `main'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/bin/srb-rbi:237:in `<main>'
```

With this change on Ruby 3:

```
Connors-MacBook-Pro-2:vglist connorshea$ CI=true bundle exec srb rbi hidden-definitions
Running command: hidden-definitions
Generating: sorbet/rbi/hidden-definitions/
Requiring all of your code
NameError: uninitialized constant ActionController::Middleware
Did you mean?  ActionController::MiddlewareStack
NameError: uninitialized constant ActiveStorage::Transformers::MiniMagickTransformer
LoadError: cannot load such file -- aws-sdk
LoadError: cannot load such file -- webrick/httputils
LoadError: cannot load such file -- graphql-rails_logger
LoadError: cannot load such file -- net-http-persistent
Did you mean?  net/http/response
LoadError: cannot load such file -- webrobots
LoadError: cannot load such file -- pundit-matchers
LoadError: cannot load such file -- rack/session/dalli
LoadError: cannot load such file -- rack-cors
LoadError: cannot load such file -- railties
LoadError: Function 'inotify_init' not found in [libc.dylib]
LoadError: cannot load such file -- rdf-aggregate-repo
LoadError: cannot load such file -- rdf-xsd
LoadError: cannot load such file -- sorbet
LoadError: cannot load such file -- sorbet-static
LoadError: cannot load such file -- spring
Did you mean?  stringio
LoadError: cannot load such file -- spring-watcher-listen
LoadError: cannot load such file -- sprockets-rails
LoadError: cannot load such file -- turbolinks-source
[250/391] require_relative './db/seeds.rb'Cleaning out database...
[251/391] require_relative './db/seeds/admin.rb'Creating admin...
[252/391] require_relative './db/seeds/advanced_users.rb'Creating Advanced Users...
[253/391] require_relative './db/seeds/companies.rb'Creating Companies...
[254/391] require_relative './db/seeds/engines.rb'Creating Engines...
[255/391] require_relative './db/seeds/game_purchases.rb'Creating Game Purchases...
[256/391] require_relative './db/seeds/games.rb'Creating Games...
[257/391] require_relative './db/seeds/genres.rb'Creating Genres...
[258/391] require_relative './db/seeds/platforms.rb'Creating Platforms...
[259/391] require_relative './db/seeds/series.rb'Creating Series...
[260/391] require_relative './db/seeds/stores.rb'Creating Stores...
[261/391] require_relative './db/seeds/users.rb'Creating Users...
[1/28] require_relative './db/seeds.rb'Cleaning out database...
[2/28] require_relative './db/seeds/games.rb'Creating Games...
Naming all Modules
Naming Object/Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/3.0.0/set/sorted_set.rb:4:in `rescue in <top (required)>': The `SortedSet` class has been extracted from the `set` library.You must use the `sorted_set` gem or other alternatives. (RuntimeError)
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/3.0.0/set/sorted_set.rb:1:in `<top (required)>'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/zeitwerk-2.4.2/lib/zeitwerk/kernel.rb:34:in `require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/real_stdlib.rb:77:in `const_get'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/real_stdlib.rb:77:in `call'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/real_stdlib.rb:77:in `real_const_get'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/constant_cache.rb:157:in `block in dfs_module'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/constant_cache.rb:145:in `each'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/constant_cache.rb:145:in `dfs_module'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/constant_cache.rb:70:in `initialize'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/gem-generator-tracepoint/tracer.rb:54:in `new'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/gem-generator-tracepoint/tracer.rb:54:in `new'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/hidden-definition-finder.rb:68:in `constant_cache'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/hidden-definition-finder.rb:75:in `all_modules_and_aliases'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/hidden-definition-finder.rb:44:in `main'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/hidden-definition-finder.rb:38:in `main'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/bin/srb-rbi:232:in `block in make_step'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/bin/srb-rbi:224:in `main'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/bin/srb-rbi:237:in `<main>'
/Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require': cannot load such file -- sorted_set (LoadError)
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:89:in `register'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:44:in `require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/zeitwerk-2.4.2/lib/zeitwerk/kernel.rb:34:in `require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/3.0.0/set/sorted_set.rb:2:in `<top (required)>'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:23:in `block in require_with_bootsnap_lfi'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/loaded_features_index.rb:92:in `register'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:22:in `require_with_bootsnap_lfi'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/bootsnap-1.5.1/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:31:in `require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/zeitwerk-2.4.2/lib/zeitwerk/kernel.rb:34:in `require'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/real_stdlib.rb:77:in `const_get'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/real_stdlib.rb:77:in `call'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/real_stdlib.rb:77:in `real_const_get'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/constant_cache.rb:157:in `block in dfs_module'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/constant_cache.rb:145:in `each'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/constant_cache.rb:145:in `dfs_module'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/constant_cache.rb:70:in `initialize'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/gem-generator-tracepoint/tracer.rb:54:in `new'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/gem-generator-tracepoint/tracer.rb:54:in `new'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/hidden-definition-finder.rb:68:in `constant_cache'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/hidden-definition-finder.rb:75:in `all_modules_and_aliases'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/hidden-definition-finder.rb:44:in `main'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/lib/hidden-definition-finder.rb:38:in `main'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/bin/srb-rbi:232:in `block in make_step'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/bin/srb-rbi:224:in `main'
	from /Users/connorshea/.rbenv/versions/3.0.0/lib/ruby/gems/3.0.0/gems/sorbet-0.5.6164/bin/srb-rbi:237:in `<main>'
```

Progress!

File: gems/sorbet/lib/gem-generator-tracepoint/tracer.rb [hunk 1]
```diff
@@ -14,6 +14,12 @@ def DelegateClass(superclass)
   end
 end
 
+# Only define the `ruby2_keywords` method if the Ruby version is less than 2.7
+# and the method isn't already defined.
+if RUBY_VERSION < '2.7' && !respond_to?(:ruby2_keywords, true)
+  def ruby2_keywords(*); end
+end
+
```
Comment (paracycle) [inline gems/sorbet/lib/gem-generator-tracepoint/tracer.rb:22]: I don't think this is a very good thing to do. We already have places in `sorber-runtime` that use `ruby2_keywords` and every usage works by checking `respond_to?`.

File: gems/sorbet/lib/gem-generator-tracepoint/tracer.rb [hunk 2]
```diff
@@ -36,7 +42,15 @@ def extend(mod, *args)
...
-        def new(*)
+        # This is a hack due to changes in kwargs with Ruby 2.7 and 3.0. Using
+        # `*` for method delegation is deprecated in Ruby 2.7 and doesn't work
+        # in Ruby 3.0.
+        # See the "compatible delegation" section in this blog post:
+        # https://www.ruby-lang.org/en/news/2019/12/12/separation-of-positional-and-keyword-arguments-in-ruby-3-0/
+        #
+        # Once Sorbet supports exclusively 2.7+ we can remove ruby2_keywords and
+        # use the `...` delegation syntax instead.
+        ruby2_keywords def new(*)
```
Comment (paracycle) [inline gems/sorbet/lib/gem-generator-tracepoint/tracer.rb:53]: I suggest that we don't change the `def` but we instead add:
```ruby
send(:ruby2_keywords, :new) if respond_to?(:ruby2_keywords, true)
```
after the method definition.

It is much more surgical and the comment can be added on top of that call.

Comment (connorshea) [reply gems/sorbet/lib/gem-generator-tracepoint/tracer.rb]: @paracycle that makes sense to me, I've updated the PR to use that pattern. 👍

**Review by paracycle — APPROVED** [review]
Looks good to me! 👍

**Review by jez — APPROVED** [review]

**Pull request merged**